wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -1493,22 +1493,23 @@ impl crate::Adapter<super::Api> for super::Adapter {
         let format_aspect = crate::FormatAspects::from(format);
         let limits = self.phd_capabilities.properties.limits;
 
-        let sample_flags = if format_aspect.contains(crate::FormatAspects::DEPTH) {
-            limits
-                .framebuffer_depth_sample_counts
-                .min(limits.sampled_image_depth_sample_counts)
+        let limits_counts = if format_aspect.contains(crate::FormatAspects::DEPTH) {
+            limits.sampled_image_depth_sample_counts
         } else if format_aspect.contains(crate::FormatAspects::STENCIL) {
-            limits
-                .framebuffer_stencil_sample_counts
-                .min(limits.sampled_image_stencil_sample_counts)
+            limits.sampled_image_stencil_sample_counts
```

Comment:
I think we still need to check `framebufferDepthSampleCounts` and `framebufferStencilSampleCounts` here.

Reply:
Oops, I misread both codes.